cli/azd/pkg/contracts/envelope.go

```diff
@@ -10,6 +10,7 @@ type EventDataType string
 const (
 	ConsoleMessageEventDataType EventDataType = "consoleMessage"
 	OperationStart              EventDataType = "operationStart"
```

Comment:
Did we miss `operationStop`? I feel it's incomplete without it -- feel free to have this as a followup.

Reply:
Yes, the action result is using just one string. I can fix that in a next PR

Reply:
https://github.com/Azure/azure-dev/issues/1272